hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieIndexConfig.java

```diff
@@ -69,6 +69,10 @@ public class HoodieIndexConfig extends HoodieConfig {
 
   private static final Logger LOG = LogManager.getLogger(HoodieIndexConfig.class);
 
+  public static final boolean DEFAULT_BLOOM_INDEX_UPDATE_PARTITION_PATH = true;
+  public static final boolean DEFAULT_SIMPLE_INDEX_UPDATE_PARTITION_PATH = true;
+
+
```

Comment:
We don't need the variables and we can assume that the default behavior reflects these.

Reply:
We'll keep the configs: `hoodie.bloom.index.update.partition.path` and `hoodie.simple.index.update.partition.path`.  Some users may still want them to be `false`.